marlowe-runtime-web/server/Language/Marlowe/Runtime/Web/Server/REST/ApiError.hs

```diff
@@ -125,7 +125,6 @@ instance ToDTO (WithdrawError 'V1) where
     WithdrawConstraintError (BalancingError _) -> ApiError "Internal error" "InternalError" Null 500
     WithdrawLoadMarloweContextFailed LoadMarloweContextErrorNotFound -> ApiError "Marlowe contract not found" "MarloweContractNotFound" Null 404
     WithdrawLoadMarloweContextFailed (LoadMarloweContextErrorVersionMismatch _) -> ApiError "Marlowe contract version mismatch" "MarloweContractVersionMismatch" Null 400
-    WithdrawLoadMarloweContextFailed (HandshakeFailed _) -> ApiError "Internal error" "InternalError" Null 500
     WithdrawLoadMarloweContextFailed LoadMarloweContextToCardanoError -> ApiError "Internal error" "InternalError" Null 500
```

Comment:
It would be helpful if all of the error messages described the actual type instead of just `"InternalError"`. Some of these messages have that detail, but others don't.

Reply:
Agreed - but definitely not in the scope of this PR